internal/innosat/source.go

```diff
@@ -0,0 +1,48 @@
+package innosat
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+//SourcePacketHeader Source Packet Header
+type SourcePacketHeader struct {
+	PacketID              uint16
+	PacketSequenceControl uint16
+	PacketLength          uint16
+}
+
+//Read SourcePacketHeader
+func (sph *SourcePacketHeader) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, sph)
```

Comment:
how about a test of a real SPH buffer from a `.rac` file resulting in the expected SPH struct?